commercetools.Sdk/commercetools.Sdk.Api/Generated/Models/Carts/CartResourceIdentifier.cs

```diff
@@ -15,5 +15,10 @@ public CartResourceIdentifier()
         {
             this.TypeId = IReferenceTypeId.FindEnum("cart");
         }
+
+        public CartResourceIdentifier(string id) : this()
+        {
+            Id = id;
+        }
```

Comment:
This is a generated file.

What do you use this constructor for? Typically you would use the init block like this:

```
new CartResourceIdentifier() { Id = id }
```